pyproject.toml

```diff
@@ -3,15 +3,79 @@ requires = [
     "setuptools",
     "packaging",
     "wheel",
-    "cython>=0.29.20; python_version>='3.10'",
-    "cython>=0.29.20,<3.0.0; python_version<='3.9'",
-    # See https://numpy.org/doc/stable/user/depending_on_numpy.html for
-    # the recommended way to build against numpy's C API:
+    "cython>=0.29.20",
     "numpy>=2.0.0",
     "scipy>=1.9",
 ]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "qutip"
+dynamic = ["version"]
+description = "QuTiP: The Quantum Toolbox in Python"
+readme = "README.md"
+requires-python = ">=3.10"
+license = {file = "LICENSE.txt"}
+keywords = ["quantum", "physics", "dynamics"]
+authors = [
+  {name = "Alexander Pitchford"},
+  {name = "Paul D. Nation"},
+  {name = "Robert J. Johansson"},
+  {name = "Chris Granade"},
+  {name = "Arne Grimsmo"},
+  {name = "Nathan Shammah"},
+  {name = "Shahnawaz Ahmed"},
+  {name = "Neill Lambert"},
+  {name = "Eric Giguère"},
+  {name = "Boxi Li"},
+  {name = "Jake Lishman"},
+  {name = "Simon Cross"},
+  {email = "qutip-admin@googlegroups.com"},
+]
```

Comment:
Is there a way to directly specify an email for the project? Having the project email address only be in the list of the authors seems not ideal.